db/seed.sql

```diff
@@ -0,0 +1,54 @@
+INSERT INTO salz.users (username, icon_path, identity_provider, identity_provider_id) VALUES 
+    ('testuser1', 'user1.png', 'fakeprovider', 'fakeid1'),
+    ('testuser2', 'user2.png', 'fakeprovider', 'fakeid2'),
+    ('testuser3', 'user3.png', 'fakeprovider', 'fakeid3'),
+    ('testuser4', 'user4.png', 'fakeprovider', 'fakeid4'),
+    ('testuser5', 'user5.png', 'fakeprovider', 'fakeid5'),
+    ('testuser6', 'user6.png', 'fakeprovider', 'fakeid6'),
+    ('testuser7', 'user7.png', 'fakeprovider', 'fakeid7'),
+    ('testuser8', 'user8.png', 'fakeprovider', 'fakeid8'),
+    ('testuser9', 'user9.png', 'fakeprovider', 'fakeid9'),
+    ('testuser10', 'user10.png', 'fakeprovider', 'fakeid10');
+
+-- Create a bot for each user
+INSERT INTO salz.bots (user_id, upload_path, status)
+SELECT id, 'bots/' || id || '.zip', 'healthy' FROM salz.users;
+
+-- Create a bunch of games for each user
+INSERT INTO salz.games (status)
+SELECT 'finished' FROM generate_series(1, 50);
+
+-- Create a bunch of game results for each game via game participants
+INSERT INTO salz.game_participants (game_id, user_id, bot_id, score, updated_at)
+with game_users as (
```

Comment:
`game_users` generates a row for every game and every user (as if every user was participating in every game) via the cross join. We use `distinct on (g.id, u.id)` because when we join to bots, it's possible that there is more than one bot for each user. The distinct on makes sure we only have one row for each `g.id, u.id`.  `random()` is a float between `0 and 1`. Note the order clause makes sure that `distinct on` only takes the most recent bot for each `g.id, u.id` pair.